src/Faithlife.Data/DbValueTypeStrategy.cs

```diff
@@ -10,5 +10,6 @@ internal enum DbValueTypeStrategy
 		Dynamic,
 		Dictionary,
 		Stream,
+		Record,
```

Comment:
"Record" isn't precisely the right term, since it is records *with positional parameters* that this PR is for. But it's fine shorthand for the internal API, and I'd eventually like a common public API (probably in [FaithlifeReflection](https://github.com/Faithlife/FaithlifeReflection)) that supports creating an arbitrary DTO with mutable properties and/or immutable constructor-initialized properties, which would probably obsolete this separate strategy.

Reply:
FWIW: https://github.com/Faithlife/FaithlifeReflection/issues/4